### DISCUSSION TITLE
Monochrome2 segmentation frames are not exported out correctly

### DISCUSSION THREAD

**Discussion by pgholami** [fo-dicom/fo-dicom#1426] [Q&A]
I tried saving the images from a Monochrome2 segmentation. What I expected was something like this (image from MicroDicom):

![test-img-00001-00130](https://user-images.githubusercontent.com/57970273/183529104-592da96c-b144-433d-b859-5689c9e23523.jpg)

But instead got a blank white image for the same frame:

![output-130](https://user-images.githubusercontent.com/57970273/183529221-90c28eec-4707-44bc-9303-e9c8f518e092.png)

This is the code I used to pull the images from the DICOM:

```
private void SaveImages(DicomDataset dataset)
  {
      var images = new DicomImage(dataset);
      for (var count = 0; count < images.NumberOfFrames; count++)
      {
          images.RenderImage(count).As<Bitmap>().Save("c:/test/images/output-" + count + ".bmp");
      }
  }
```

I also tried reading the pixel values for each frame and the values all returned white as well. I'm not quite sure what I'm doing wrong or if this is a bug with fo-dicom. For reference, I have included a sample copy of the DICOM I'm using:

[sample.zip](https://github.com/fo-dicom/fo-dicom/files/9285376/sample.zip)

**Comment by gofal** [comment]
oh, this file has 1 bits per pixel. I am affraid fo-dicom does not handle images with less than 8 bits per pixel correctly.
